compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -2752,6 +2752,9 @@ object SymDenotations {
     /** Sets all missing fields of given denotation */
     def complete(denot: SymDenotation)(using Context): Unit
 
+    /** Is this a completer for an explicit type tree */
+    def isNonInfering: Boolean = false
```

Comment:
I think inferring takes two "r"? Also, wouldn't "isExplicit" be more straightforward?

Reply:
Sure, changed to `isExplicit`